content/docs/getting-started/hello-world.md

```diff
@@ -71,93 +56,74 @@ Next, we'll add some to the `client` `TcpStream`. This asynchronous task now cre
 the stream and then yields it once it's been created for additional processing.
 
 ```rust
-# #![deny(deprecated)]
-# extern crate tokio;
-#
-# use tokio::net::TcpStream;
-# use tokio::prelude::*;
-# fn main() {
-# let addr = "127.0.0.1:6142".parse().unwrap();
-let client = TcpStream::connect(&addr).and_then(|stream| {
-    println!("created stream");
-
-    // Process stream here.
-
-    Ok(())
-})
-.map_err(|err| {
-    // All tasks must have an `Error` type of `()`. This forces error
-    // handling and helps avoid silencing failures.
-    //
-    // In our example, we are only going to log the error to STDOUT.
-    println!("connection error = {:?}", err);
-});
-# }
+use tokio::net::TcpStream;
+
+fn main() {
+    let addr = "127.0.0.1:6142";
+    let client = TcpStream::connect(addr);
+
+    let future = async move {
+        match client.await {
+            Ok(_stream) => {
+                println!("created stream");
+
+                // Process stream here.
+            }
+            Err(err) => {
+                // In our example, we are only going to log the error to STDOUT.
+                println!("connection error = {:?}", err);
+            }
+        };
+    };
+}
 ```
 
-The call to `TcpStream::connect` returns a [`Future`] of the created TCP stream.
+Inside the block the call to `TcpStream::connect` returns a [`Future`] of the created TCP stream.
 We'll learn more about [`Futures`] later in the guide, but for now you can think of
 a [`Future`] as a value that represents something that will eventually happen in the
 future (in this case the stream will be created). This means that `TcpStream::connect` does
 not wait for the stream to be created before it returns. Rather it returns immediately
```

Comment:
Also, I know this was already here, but I think there should be a comma after "Rather"
```suggestion
not wait for the stream to be created before it returns. Rather, it returns immediately
```